subworkflows/local/bgc.nf

```diff
@@ -80,6 +80,9 @@ workflow BGC {
                                                 meta, files ->
                                                 [meta, files.flatten()]
                                             }
+        ANTISMASH_ANTISMASHLITE.out.gbk_results.ifEmpty(
+            ch_antismashresults_for_combgc = Channel.empty()
+        )
```

Comment:
This is a bit ugly to me Nextflow syntax-wise (setting a channel inside an operator), but if it works I'll let it slide at the moment.

Did `ch_antismashresults_for_combgc =  ANTISMASH_ANTISMASHLITE.out.gbk_results.ifEmpty([])` not work?

Reply:
Unfortunately not. That is because the output channel is accessed (causing error) before executing the `.ifEmpty([])`.

Reply:
Fixed, see update in [2970101](https://github.com/nf-core/funcscan/pull/386/commits/29701018c65a1694a96c4196af6dae5801529d0e)